### PR TITLE
metrics format change

### PR DESCRIPTION
**Pull request by richardstartin**

### CODE REVIEW

**Review by tylerbenson — APPROVED** [review]

File: internal-api/src/main/java/datadog/trace/api/Config.java [hunk 1]
```diff
@@ -733,7 +733,10 @@ private Config(final String runtimeId, final ConfigProvider configProvider) {
   }
 
   public WellKnownTags getWellKnownTags() {
-    return new WellKnownTags(getHostName(), tags.get(ENV), serviceName, tags.get(VERSION));
+    CharSequence env = tags.get(ENV);
+    CharSequence version = tags.get(ENV);
```
Comment (mcculls) [inline internal-api/src/main/java/datadog/trace/api/Config.java:737]: version != ENV ?

Comment (richardstartin) [reply internal-api/src/main/java/datadog/trace/api/Config.java:737]: Thanks, adding a test!

File: dd-trace-core/src/main/java/datadog/trace/common/metrics/SerializingMetricWriter.java [hunk 2]
```diff
@@ -67,34 +70,35 @@ public void startBucket(int metricCount, long start, long duration) {
 
   @Override
   public void add(MetricKey key, AggregateMetric aggregate) {
-    writer.startMap(9);
+    writer.startMap(7);
```
Comment (mcculls) [inline dd-trace-core/src/main/java/datadog/trace/common/metrics/SerializingMetricWriter.java:73]: I'm counting 9 elements below - is this change right or am I missing something?

Comment (richardstartin) [reply dd-trace-core/src/main/java/datadog/trace/common/metrics/SerializingMetricWriter.java:73]: You're right. This is the problem with a test which mirrors the implementation (it incorrectly checks that there is a 7 element map, then correctly validates all 9 elements). The integration test would have caught this if I had an image for the latest tracer yet.

Comment (richardstartin) [reply dd-trace-core/src/main/java/datadog/trace/common/metrics/SerializingMetricWriter.java]: Added a consistency check to the test to catch this sort of thing next time.

**Review by mcculls — APPROVED** [review]

**Review by devinsba — APPROVED** [review]

**Pull request merged**